custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2600,18 +2600,6 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         return i;
     };
 
-    vm.moveToLocation = function (loc, index) {
```

Comment:
2 down, 3 to go of 100% copy/pasted declarations of this method.

Reply:
Keep fighting the good fight